broadcast/internal/acceptance_tests/arc_submit_tx_test.go

```diff
@@ -33,6 +33,7 @@ var successfulSubmitBatchResponse = `
 ]
 `
 
+// TestSubmitTransaction tests submitting a transaction to the ARC API.
```

Comment:
Will tests and internal code also appear in godocs?

Reply:
Discussed internally - resolved.